presto-mysql/src/main/java/io/prestosql/plugin/mysql/MySqlClient.java

```diff
@@ -365,6 +365,39 @@ public boolean isLimitGuaranteed(ConnectorSession session)
         return true;
     }
 
+    @Override
+    public void setColumnComment(JdbcIdentity identity, JdbcTableHandle handle, JdbcColumnHandle column, Optional<String> comment)
+    {
+        try (Connection connection = connectionFactory.openConnection(identity)) {
+            ResultSet resultSet = getColumns(handle, connection.getMetaData());
+
+            String colDefintion = "";
+            while (resultSet.next()) {
+                if (resultSet.getString("COLUMN_NAME").equalsIgnoreCase(column.getColumnName())) {
+                    colDefintion = format("%s %s(%s)%s%s%s",
+                            column.getColumnName(),
+                            column.getJdbcTypeHandle().getJdbcTypeName().get(),
+                            column.getJdbcTypeHandle().getColumnSize(),
+                            resultSet.getString("IS_NULLABLE").equalsIgnoreCase("YES") ? "" : " NOT NULL",
+                            resultSet.getString("IS_AUTOINCREMENT").equalsIgnoreCase("NO") ? "" : " AUTO_INCREMENT",
+                            resultSet.getString("COLUMN_DEF") == null ? "" : format(" DEFAULT '%s'", resultSet.getString("COLUMN_DEF")));
+                    break;
+                }
+            }
+            resultSet.close();
+
+            String sql = format(
+                    "ALTER TABLE %s MODIFY %s COMMENT %s",
+                    quoted(handle.getRemoteTableName()),
+                    colDefintion,
+                    comment.isPresent() ? format("'%s'", comment.get()) : "''");
```

Comment:
We can simplify as `"ALTER TABLE %s MODIFY %s COMMENT '%s'"` and `comment.orElse("")`